src/utils.js

```diff
@@ -79,14 +79,14 @@ export function booleanGetter(element, attributeName) {
  * @param {number} value
  */
 export function intSetter(element, attributeName, value) {
-  element.setAttribute(attributeName, value);
+  element.setAttribute(attributeName, `${value}`);
 }
 
 /**
  * Standard getter for a Custom Element int property reflected to attribute.
  * @param {HTMLElement} element
  * @param {string} attributeName
- * @param {string} [defaultValue=0]
+ * @param {number|string} [defaultValue=0]
```

Comment:
Do you think this should be always a number actually?

Reply:
I do yeah